### PR TITLE
Parking options

### PR DESCRIPTION
**Pull request by herlitze**

### CODE REVIEW

**Review by ue71603 — DISMISSED** [review]

File: OJP/OJP_PlaceSupport.xsd [hunk 1]
```diff
@@ -267,6 +272,21 @@
 			</xs:element>
 		</xs:choice>
 	</xs:complexType>
+	<xs:complexType name="PointOfInterestAdditionalInformationStructure">
+		<xs:annotation>
+			<xs:documentation>Generic structure for additional information on POIs.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="Category" type="xs:string" minOccurs="0"/>
+			<xs:element name="Attributes" type="KeyValueType" maxOccurs="unbounded"/>
```
Comment (Aurige) [inline OJP/OJP_PlaceSupport.xsd:281]: Already unbounded inside the PointOfInterest->AdditionalInformation ... do we really need 2 unbounded levels

File: OJP/OJP_PlaceSupport.xsd [hunk 2]
```diff
@@ -248,6 +248,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:element ref="TopographicPlaceRef" minOccurs="0"/>
+			<xs:element name="AdditionalInformation" type="PointOfInterestAdditionalInformationStructure" minOccurs="0" maxOccurs="unbounded">
```
Comment (Aurige) [inline OJP/OJP_PlaceSupport.xsd:251]: unbounded so needs a wrapper ....

File: OJP/OJP_PlaceSupport.xsd (hunk 1 above)
Comment (ue71603) [reply OJP/OJP_PlaceSupport.xsd:281]: rewritenn

File: OJP/OJP_PlaceSupport.xsd (hunk 2 above)
Comment (ue71603) [reply OJP/OJP_PlaceSupport.xsd:251]: rewritten

**Review by trurlurl — DISMISSED** [review]

**Review by ue71603 — APPROVED** [review]

File: OJP/OJP_PlaceSupport.xsd [hunk 3]
```diff
@@ -248,6 +248,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:element ref="TopographicPlaceRef" minOccurs="0"/>
+			<xs:element name="POIAdditionalInformation" type="PointOfInterestAdditionalInformationStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Additional information for this POI. (E. g. information on available parking slots, charging infrastructures, sharing vehicles.)</xs:documentation>
```
Comment (skinkie) [inline OJP/OJP_PlaceSupport.xsd:253]:
```suggestion
					<xs:documentation>Additional information for this POI. (E.g. information on available parking slots, charging infrastructures, sharing vehicles.)</xs:documentation>
```

**Review by skinkie — DISMISSED** [review]

File: OJP/OJP_Places.xsd [hunk 4]
```diff
@@ -204,6 +204,7 @@
 					<xs:documentation>Filter to narrow down POI searches.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="Extension" type="xs:anyType" minOccurs="0"/>
```
[the remark this answers is not in the document]
Comment (herlitze) [reply OJP/OJP_Places.xsd]: I added the extension to give more options for filtering. I had filters e.g. for power, voltage, plugtype, ... for charging in mind. There will be several options for sharing and parking as well.

[the remark this answers is not in the document]
Comment (ue71603) [reply OJP/OJP_Places.xsd]: I will add a to be discussed flag to this PR

**Review by ue71603 — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Review by ue71603 — DISMISSED** [review]

**Review by skinkie — APPROVED** [review]

**Review by sgrossberndt — APPROVED** [review]

**Pull request merged**